index.js

```diff
@@ -217,23 +217,32 @@ function updateDesigns(designPaths, callback) {
         throw new VError(msg);
     }
     // everything looks good, we're ready to update any designs that need it
+    var i = 0;
```

Comment:
This variable and everything that goes with it is ambiguous.
At the very least it needs a more descriptive name.
Possibly, a different approach to solving whatever problem is being solved should be taken.